src/software/abinit.jl

```diff
@@ -962,3 +962,29 @@ function write_abinit_modes(modes::Array{SVector{6,Float64}}, energies::Vector{F
         end
     end
 end
+
+"""
+    read_abinit_anaddb_PHDOS(filename::AbstractString) -> Tuple{DensityOfStates, Vector{ProjectedDensityOfStates}}
+Reads the PHDOS file from ABINIT's anaddb script.
```

Comment:
I'd recommend explaining a bit more about the PHDOS file - what it contains, perhaps anything notable about its format.

Reply:
Added! Let me know if it's sufficient. I didn't want to add too much because the header of the PHDOS file is pretty straightforward.